mpReview.py

```diff
@@ -325,6 +325,26 @@ def setupDatabaseSelectionView(self):
     self.databaseGroupBox.setLayout(databaseGroupBoxLayout)
     self.databaseSelectionWidgetLayout.addWidget(self.databaseGroupBox, 3, 0, 1, 3)
     
+    ###############################
+    ### Select terminology file ### 
+    ###############################
+    
+    self.terminologyGroupBox = qt.QGroupBox("Terminology")
+    terminologyGroupBoxLayout = qt.QFormLayout()
+    self.selectTerminologyFileButton = qt.QPushButton("Select Segmentation JSON Terminology file")
+    self.selectTerminologyFileButton.setEnabled(True)
+    terminologyGroupBoxLayout.addRow(self.selectTerminologyFileButton)
+    
+    self.terminologyGroupBox.setLayout(terminologyGroupBoxLayout)
+    self.databaseSelectionWidgetLayout.addWidget(self.terminologyGroupBox, 4, 0, 1, 3)
+    
+  def selectTerminologyFile(self):
+    """ open a QFileDialog box where the user can optionally choose a segmentation JSON terminology file """
+    segJsonPrompt = qt.QFileDialog()
+    self.terminologyFile = segJsonPrompt.getOpenFileName(None,"Select File", "", "All Files (*);;Python Files (*.json)")
```

Comment:
Should "Python files" be "JSON files"?

Reply:
This is now changed as I am using ctkPathLineEdit -- I did the following to make sure only json files are displayed when choosing the terminology file: `self.terminologyFilePathLineEdit.nameFilters = ['*.json']`